### PR TITLE
Spanish Localization added

### PR DESCRIPTION
**Pull request by tillmanj**
Spanish (Spain) translations added for inclusion.

### CODE REVIEW

File: errors.toml [hunk 1]
```diff
@@ -46,6 +51,13 @@ future to receive valid responses. For tips on how, come have a chat \
 with us in #esi on tweetfleet slack. If you're not on tweetfleet \
 slack yet, you can get an invite here -> \
 https://www.fuzzwork.co.uk/tweetfleet-slack-invites/"""
+es-es = """
+Este software ha superado el límite de error para ESI. Si \
+usted es un usuario, póngase en contacto con el desarrollador de este software. Si \
+usted es un desarrollador / mantenedor, por favor haga un mayor esfuerzo en el \
+futuro para recibir respuestas válidas. Para consejos sobre cómo hacerlo, chatee \
+con nosotros en #esi en tweetfleet Slack. Si no está en tweetfleet \, puede obtener una invitación aquí -> \
```
Comment (ghost) [inline errors.toml]: Escaping is not quite correct here, no space between tweetfleet and the comma, backslash newline needs to be between the comma and puede. Should be like this:
`...tweetfleet, \`
`puede...`

File: errors.toml [hunk 2]
```diff
@@ -105,6 +120,7 @@ zh = "未提供身份验证信息"
 # ErrorAuth is used in SSO communication failures
 de = "authentifizierungsfehler"
 en-us = "authentication failure"
+es-es = "fallo de autentificacion"
```
Comment (ghost) [inline errors.toml:124]: I'm told this should be 'autentificación'

**Review by ghost — CHANGES_REQUESTED** [review]
Two changes needed. I'd do them myself but I can't seem to add a new commit to the PR from the github UI :crying_cat_face:

File: errors.toml (hunk 1 above)
Comment (tillmanj) [reply errors.toml]: fixed.

File: errors.toml (hunk 2 above)
Comment (tillmanj) [reply errors.toml:124]: fixed.

**Review by ghost — APPROVED** [review]

**Pull request merged**